vuepress/docs/next/tutorials/create/pb/publish-project-bundle.md

```diff
@@ -1,165 +1,56 @@
 ---
 sidebarDepth: 2
 ---
-# Build and Publish a Project Bundle
+# Build and Publish a Bundle Project
 ## Overview
-This tutorial demonstrates how to deploy an existing Entando project directory into the [Local Hub](../../../docs/getting-started/concepts-overview.md#local-hub). This involves building a Docker image from your microservice, creating an Entando Bundle, checking your Bundle artifacts into Git, and deploying the Bundle into Kubernetes.
+This tutorial demonstrates how to deploy an existing Entando project directory into the [Local Hub](../../../docs/getting-started/concepts-overview.md#local-hub). The process involves: 
```

Comment:
involves>> includes ??

Reply:
"includes" is not exhaustive, whereas "involves" seems more representative of everything; i.e. "includes" is "some but not limited to" while "involves" approaches all-encompassing. regardless, i was not happy with the word choice